lib/commands/db.js

```diff
@@ -51,7 +51,7 @@ function executeDB (internals, config, callback) {
 
           db.close();
           if (typeof callback === 'function') callback(err);
-          process.exit(1);
+          process.exit(err ? 1 : 0);
```

Comment:
This should move into the else branch of the previous if clause as it should not exit if callback is provided. If I mind right that will also close one other issue.

Reply:
Sorry I didn't answer this on time before the stale-bot closed this PR. I see that #576 takes care of this.